### DISCUSSION TITLE
Welcome to vftool Discussions!

### DISCUSSION THREAD

**Discussion by evansm7** [evansm7/vftool#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by praneet-suresh** [comment]
Hey, hope you're doing good! Great work with Vftool. I've something to share, here's my repo of a GUI alternate for Vftool https://github.com/PraneetNeuro/Project-Mendacius with certain additional features, do check it out and let's keep it going! Cheers!
